### ISSUE TITLE
Template-bench causing errors

### ISSUE THREAD

**Issue by stephenbe** [10Dimensional/hugo-algolia#1]
Hello,

Thank you for making this plugin. I am trying to create the json file, but it seems to be failing when I call the script...This is what I get : 

```
C:\Projets\gabarit-site-hugo>npm run index

> template-bench@0.0.1 index C:\Projets\gabarit-site-hugo
> hugo-algolia -i "content/**" -o public/my-index.json

C:\Projets\gabarit-site-hugo\node_modules\hugo-algolia\lib\utils.js:11
async function copySynonyms(fromIndex, toIndex) {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Projets\gabarit-site-hugo\node_modules\hugo-algolia\lib\index.js:11:52)
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "index"
npm ERR! node v6.11.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! template-bench@0.0.1 index: `hugo-algolia -i "content/**" -o public/my-index.json`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the template-bench@0.0.1 index script 'hugo-algolia -i "content/**" -o public/my-index.json'.
```

**Comment by 10Dimensional** [comment]
Heyo, sorry for the late reply. I'll take a look at this and get back to you! Thanks for letting me know.

**Comment by tovare** [comment]
I got this error message but resolved it by using a later version of node (v8.9.4)

nvm install v8.9.4
